alacritty/src/event.rs

```diff
@@ -1044,8 +1044,9 @@ impl input::Processor<EventProxy, ActionContext<'_, Notifier, EventProxy>> {
                         }
                     },
                     TerminalEvent::ResetTitle => {
-                        if self.ctx.config.window.dynamic_title {
-                            self.ctx.display.window.set_title(&self.ctx.config.window.title);
+                        let window_config = &self.ctx.config.window;
+                        if window_config.dynamic_title {
```

Comment:
```suggestion
                        if window_config.dynamic_title {
                            let window_config = &self.ctx.config.window;
```

Reply:
I don't see a reason to do so? I mean it's the same amount of code and you access window_config in `if` anyway, so you can extract it and use in 2 places instead of just one.

Reply:
Didn't see that you actually use it in the conditional, thought it was just inside. All good. :+1: